dynet/lstm.h

```diff
@@ -45,12 +45,20 @@ struct LSTMBuilder : public RNNBuilder {
   Expression set_h_impl(int prev, const std::vector<Expression>& h_new) override;
   Expression set_s_impl(int prev, const std::vector<Expression>& s_new) override;
  public:
+  bool gal_dropout;
```

Comment:
Maybe I missed this, but it looks like this isn't initialized to a default value anywhere?